tests/graph/test_op.py

```diff
@@ -232,3 +232,26 @@ def perform(self, *_):
 
     x = pt.TensorType(dtype="float64", shape=(1,))("x")
     assert SomeOp()(x).type == pt.dvector
+
+
+def test_op_name():
+    x = pt.vector("x")
+    y = pt.vector("y")
+    op_name = "op_name"
+
+    class MultiOutOp(Op):
```

Comment:
Also test with (and without) default output and you should have all cases covered